[2024-01-18T21:57:14.657+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_theta scheduled__2024-01-01T00:30:00+00:00 [queued]>
[2024-01-18T21:57:14.679+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_theta scheduled__2024-01-01T00:30:00+00:00 [queued]>
[2024-01-18T21:57:14.680+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T21:57:14.681+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-18T21:57:14.681+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T21:57:14.714+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): test_model_theta> on 2024-01-01 00:30:00+00:00
[2024-01-18T21:57:14.726+0300] {standard_task_runner.py:55} INFO - Started process 5311 to run task
[2024-01-18T21:57:14.746+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'test_model_theta', 'scheduled__2024-01-01T00:30:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmp0zbcym2n']
[2024-01-18T21:57:14.749+0300] {standard_task_runner.py:83} INFO - Job 10: Subtask test_model_theta
[2024-01-18T21:57:14.907+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.test_model_theta scheduled__2024-01-01T00:30:00+00:00 [running]> on host xflow
[2024-01-18T21:57:15.075+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=test_model_theta
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T00:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T00:30:00+00:00
[2024-01-18T21:57:15.081+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-18T21:57:15.082+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/angry/mlops_4/scripts/test_model.py theta']
[2024-01-18T21:57:15.114+0300] {subprocess.py:86} INFO - Output:
[2024-01-18T21:57:22.440+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-18T21:57:22.498+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=test_model_theta, execution_date=20240101T003000, start_date=20240118T185714, end_date=20240118T185722
[2024-01-18T21:57:22.566+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-18T21:57:22.599+0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
